From 6489bd907c8caf087ed6943ad7b3e9767b96460b Mon Sep 17 00:00:00 2001
From: Scott Talbert <swt@techie.net>
Date: Thu, 4 Jun 2020 10:46:28 -0400
Subject: [PATCH] Add some missing private assignment operators

This is needed as part of fixing #1636 as a new version of SIP is required,
which makes these changes necessary.

(cherry picked from commit 676b95ad1752eb2180e48e1b382ec06b0d28d5e7)
---
 etg/filesys.py  | 1 +
 etg/renderer.py | 6 ++++++
 2 files changed, 7 insertions(+)

diff --git a/etg/filesys.py b/etg/filesys.py
index 57915dd7..27625d4c 100644
--- a/etg/filesys.py
+++ b/etg/filesys.py
@@ -40,6 +40,7 @@ def run():
     c = module.find('wxFileSystem')
     assert isinstance(c, etgtools.ClassDef)
     c.addPrivateCopyCtor()
+    c.addPrivateAssignOp()
     c.find('AddHandler.handler').transfer = True
     c.find('RemoveHandler').transferBack = True
 
diff --git a/etg/renderer.py b/etg/renderer.py
index b87d6952..0afb0045 100644
--- a/etg/renderer.py
+++ b/etg/renderer.py
@@ -66,6 +66,12 @@ def run():
             #endif
         """)
 
+    c = module.find('wxSplitterRenderParams')
+    c.addPrivateAssignOp()
+
+    c = module.find('wxRendererVersion')
+    c.addPrivateAssignOp()
+
     #-----------------------------------------------------------------
     tools.doCommonTweaks(module)
     tools.runGenerators(module)
-- 
2.35.2.windows.1

